brave/browser-laptop
# restyle preferences payments

Fix #5494

Auditors @bsclifton @bradleyrichter 

Test Plan:

1. Open Brave
2. Go to Preferences > Payments
3. Toggle a site to not be included
4. Toggle hide excluded sites and make sure that one disappears
5. Make sure the general design looks like the mock in #5494
6. Make sure clicking the remove button for a site removes it from the list

Here is a screenshot of this branch:
![screen shot 2016-12-08 at 2 20 43 pm](https://cloud.githubusercontent.com/assets/490294/21030134/d7704d00-bd51-11e6-94a5-1c3346c8366f.png)

Spec image:
<img width="801" alt="screenshot 2016-12-17 17 31 04" src="https://cloud.githubusercontent.com/assets/3362943/21285540/ad422b96-c47e-11e6-9da1-8408547ad4d7.png">
